compact/mrich.xml

```diff
@@ -3,7 +3,7 @@
   <define>
     <constant name="MRICH_rmin"   value="8*cm"/>
     <constant name="MRICH_rmax"   value="min(63*cm, BackwardPIDRegion_rmax)"/>
-    <constant name="MRICH_length" value="BackwardPIDRegion_length"/>
+    <constant name="MRICH_length" value="BackwardPIDRegion_length-28.87*cm"/>
```

Comment:
This I don't understand.

Reply:
This shifts the mrich back by the difference in length between the mrich and the pfrich to ensure they are both placed at the back of the PID region. Not sure how else to do this, except to maybe introduce an extra parameter in the top level definitions  file (which would be a bit cleaner)

Reply:
I should double-check the `dawn` views again, but I think we resolved the pfRICH/mRICH placement in #127 (just look at `definitions.xml` and `{pf,m}rich.xml`, there are several workaround/test commits in that PR that need to be reverted)

Reply:
> This shifts the mrich back by the difference in length between the mrich and the pfrich

That's where I am confused: the difference is 11.4 cm (pfRICH 54.1 cm - mRICH 42.7 cm), not 28.87 cm.

Reply:
Let's resolve this in #158

Reply:
Really? I'm just done here.